mesos_slave/datadog_checks/mesos_slave/mesos_slave.py

```diff
@@ -120,37 +119,36 @@ def check(self, instance):
             raise ConfigurationError('Mesos instance missing "url" value.')
 
         url = instance['url']
-        tags = instance.get('tags', [])
+        tags = list(instance.get('tags', []))
         tasks = instance.get('tasks', [])
         master_port = instance.get("master_port", DEFAULT_MASTER_PORT)
 
-        try:
-            self._process_state_info(url, tasks, master_port, tags)
-            self._process_stats_info(url, tags)
-        except CheckException as e:
-            self.log.error("Error running check mesos_slave with exception %s", e)
-            raise
+        self._process_state_info(url, tasks, master_port, tags)
+        self._process_stats_info(url, tags)
 
     def _process_state_info(self, url, tasks, master_port, tags):
-        tags = set(tags)
+        state_tags = list(tags)
```

Comment:
why this change? Do we allow duplicate tags?

Reply:
Not really, but we don't need it either.